modules/logging/filters.go

```diff
@@ -185,15 +187,113 @@ func (m IPMaskFilter) Filter(in zapcore.Field) zapcore.Field {
 	return in
 }
 
+type ActionType string
+
+const (
+	ReplaceType ActionType = "replace"
+	DeleteType  ActionType = "delete"
+)
```

Comment:
Seems like this is code in common with https://github.com/caddyserver/caddy/pull/4424 so I suppose whichever is merged first will need to be rebased to adjust afterwards.

Reply:
indeed